rotkehlchen/api/v1/resources.py

```diff
@@ -1837,6 +1837,10 @@ def post(self, asset: Asset, file: FileStorage) -> Response:
 
         return response
 
+    @use_kwargs(get_schema, location='view_args')
```

Comment:
Sounds strange to use `get_schema` for a `delete` request. Maybe rename it to something more general?